crypto/encrypt_test.go

```diff
@@ -141,3 +141,69 @@ func TestTwofishEntropy(t *testing.T) {
 		t.Error("supposedly high entropy ciphertext has been compressed!")
 	}
 }
+
+// TestUnitCiphertextUnmarshalJSON tests that Ciphertext.UnmarshalJSON
+// correctly fails on invalid JSON marshalled Ciphertext.
+func TestUnitCiphertextUnmarshalJSON(t *testing.T) {
```

Comment:
name is not complete, because this test is only about invalid json.

It's okay for the test function names to be long, it's better to be clear about what is being tested.